corehq/apps/users/bulkupload.py

```diff
@@ -711,7 +711,14 @@ def _get_group_rows():
     return group_headers, _get_group_rows()
 
 
-def dump_users_and_groups(domain, download_id, user_filters):
+def count_users_and_groups(domain, user_filters, group_memoizer):
+    users_count = get_commcare_users_by_filters(domain, user_filters, count_only=True)
+    groups_count = len(group_memoizer.groups)
+
+    return users_count + groups_count
```

Comment:
I see groups is accounted for here in the total, but it looks like the progress isn't being updated for groups.

Reply:
Yeah, I'm faking this a bit...
The user's loop actually also, confusingly, handles groups. The `parse_group` method just reformats the dicts that were pulled previously. I figured that even in the worse case (1000s of groups) this would complete faster than the progress poll happens on the front end, so it didn't seem worth it to me to include the overhead of updating the celery task progress in that loop. 
I added the group count here to make the UI more realistic, but I could remove that if you think it would be confusing.

Reply:
That makes sense.  I imagine the number of users would dwarf the number of groups in any event, so the time difference is probably minimal.  Was just thinking that we'd see steady progress from say, 0 to 90%, then an immediate jump from 90 to 100.  That also seems fine, and would account for any delays that happen after the iterative processing.  I don't really have a sense of where the time breakdown is in here anyways, so I agree that it makes sense to just push this out and not worry too much about getting it perfect.